xds/src/main/java/io/grpc/xds/ClusterImplLoadBalancer.java

```diff
@@ -77,10 +79,9 @@ final class ClusterImplLoadBalancer extends LoadBalancer {
       Strings.isNullOrEmpty(System.getenv("GRPC_XDS_EXPERIMENTAL_CIRCUIT_BREAKING"))
           || Boolean.parseBoolean(System.getenv("GRPC_XDS_EXPERIMENTAL_CIRCUIT_BREAKING"));
 
-  private static final Attributes.Key<ClusterLocalityStats> ATTR_CLUSTER_LOCALITY_STATS =
-      Attributes.Key.create("io.grpc.xds.ClusterImplLoadBalancer.clusterLocalityStats");
-  private static final Attributes.Key<String> ATTR_CLUSTER_LOCALITY_NAME =
-      Attributes.Key.create("io.grpc.xds.ClusterImplLoadBalancer.clusterLocalityName");
+  private static final Attributes.Key<AtomicReference<ClusterLocality>>
+      ATTR_CLUSTER_LOCALITY =
```

Comment:
You can put ATTR_CLUSTER_LOCALITY on the same line as the type.

Reply:
Done